roles/ansible-runner/tasks/runner.yml

```diff
@@ -33,6 +33,12 @@
   secret_file_path: "{{ ansible_runner_secrets_path }}"
   secret_file_user: "{{ item.user }}"
 
+- name: Disable cron job
+  copy:
```

Comment:
The way i wanted to use this was to do -e ansible_runner_disabled=True from the CLI to turn everything off in development.